.devcontainer/docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+services:
+  web:
+    # Uncomment if you want to override the service's Dockerfile to one in the .devcontainer 
+    # folder. Note that the path of the Dockerfile and context is relative to the *primary* 
+    # docker-compose.yml file (the first in the devcontainer.json "dockerComposeFile"
+    # array). The sample below assumes your primary file is in the root of your project.
+    #
+    # build:
+    #   context: .
+    #   dockerfile: .devcontainer/Dockerfile
+
+    volumes:
+      # Update this to wherever you want VS Code to mount the folder of your project
+      - ..:/workspaces:cached
+    environment:
+      SELENIUM_BROWSER: remote
+      SELENIUM_BROWSER_HOST: http://chrome-server:4444
+      HEADLESS: true
+
+    # Overrides default command so things don't shut down after the process ends.
+    command: sleep infinity
+
+  chrome-server:
+    image: seleniarm/standalone-chromium
+    ports:
+      - 4444:4444
+      - 7900:7900
+      - 5900:5900
```

Comment:
This file is duplicating what we have in our `docker-compose.test.yml` file. I think we should try using this setup, and if people feels good about it, we can get rid of that test compose entirely in favor of devcontainers. For now, I'd rater keep both just in case